world-bank-app/src/index.css

```diff
@@ -1,13 +1,56 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+		"Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+		"Helvetica Neue", sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+		monospace;
+}
+/* .custom-year-range-picker-container {
+	color: "black";
+} */
+.custom-year-range-picker .yrp-picker-box {
+	height: 40px;
+	width: 200px;
+	border-radius: 5px;
+	background-color: white;
+	color: black;
+	position: absolute;
+}
+
+.custom-year-range-picker .yrp-picker-box span {
+	color: black;
+
+	font-size: medium;
+}
+
+.custom-year-range-picker .yrp-picker-box i {
+	border-color: gray;
+}
+
+.selection-bar {
+	padding-left: 10%;
+	padding-right: 10%;
+	height: 85px;
+	width: 100%;
+	display: flex;
+	justify-content: space-evenly;
+}
+
+.bar-item {
+	width: 150px;
+	height: 40px;
+}
+
+.search-button {
+	align-self: flex-end;
+}
+
+.year-range-container {
+	text-align: center;
 }
```

Comment:
https://getbootstrap.com/docs/5.0/utilities/spacing/
Using these bootstrap classNames might make spacing easier than doing in CSS manually.
https://getbootstrap.com/docs/5.0/utilities/text/
Same for aligning text